### PR TITLE
Orderless configuration

### PR DESCRIPTION
**Pull request by sowmyadn010501**
updated orderless config like below ospfv3 neighbor at depth 1.
```

- ospfv3 neighbor FE80::AC5 priority 100 poll-interval 100 cost 100
- ospfv3 neighbor FE80::AC4 priority 1 poll-interval 1 database-filter all out
```
PASS LOG: https://earms-trade.cisco.com/tradeui/logs/logfile?logfile=TaskLog.Task-1&begin=1414611&size=1958240&archive=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr%2Fusers%2Fvijviswa%2Farchive%2F24-06%2Fcli2show_job.2024Jun18_10:16:00.697250.zip&ats=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr&submitter=vijviswa&from=trade&view=all&atstype=pyATS
updated orderless config like below ospfv3 1 ipv4 neighbor at depth 1.
```
20233:  -   ospfv3 1 ipv4 neighbor FE80::AC5 priority 100 poll-interval 100 cost 100
20234:  -   ospfv3 1 ipv4 neighbor FE80::AC4 priority 1 poll-interval 1 database-filter all out
```
PASS LOG: https://earms-trade.cisco.com/tradeui/logs/logfile?logfile=TaskLog.Task-1&begin=24490370&size=848576&archive=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr%2Fusers%2Fvijviswa%2Farchive%2F24-06%2Fcli2show_job.2024Jun19_08:53:05.637932.zip&ats=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr&submitter=vijviswa&from=trade&view=all&atstype=pyATS
updated orderless config like below ospfv3 1 ipv6 neighbor at depth 1.
```
275696:  -   ospfv3 1 ipv6 neighbor FE80::AC5 priority 100 poll-interval 100 cost 100
275697:  -   ospfv3 1 ipv6 neighbor FE80::AC4 priority 1 poll-interval 1 database-filter all out
```
PASS LOG:https://earms-trade.cisco.com/tradeui/logs/logfile?logfile=TaskLog.Task-1&begin=27157921&size=853292&archive=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr%2Fusers%2Fvijviswa%2Farchive%2F24-06%2Fcli2show_job.2024Jun19_08:53:05.637932.zip&ats=%2Fnobackup%2Fvijviswa%2Fcmt_script%2Fpolaris%2Fbinos%2Flinkfarm%2Fsdks-host%2Fiosxe-pyats%2Fnativesdk%2Fsdk%2Fsysroots%2Fx86_64-xesdk-linux%2Fusr&submitter=vijviswa&from=trade&view=all&atstype=pyATS

### CODE REVIEW

**Comment by sowmyadn010501** [comment]
I raised this PR for the same issue: https://github.com/CiscoTestAutomation/yang/pull/136 but in different PR as the user is different

File: ncdiff/src/yang/ncdiff/runningconfig.py [hunk 1]
```diff
@@ -100,6 +100,9 @@
     (re.compile(r'^ *distribute-list '), 2),
     (re.compile(r'^ *area '), 1),
     (re.compile(r'^ *ip ospf message-digest-key '), 1),
+    (re.compile(r'^ *ospfv3 neighbor '), 1),
+    (re.compile(r'^ *ospfv3 1 ipv(\d) neighbor'), 1),
```
Comment (yuekyang) [inline ncdiff/src/yang/ncdiff/runningconfig.py:104]: Could this line be:
`    (re.compile(r'^ *ospfv3 \d+ ipv(\d) neighbor'), 1),`

File: ncdiff/src/yang/ncdiff/runningconfig.py [hunk 2]
```diff
@@ -100,6 +100,9 @@
...
+    (re.compile(r'^ *ospfv3 1 neighbor '), 1),
```
Comment (yuekyang) [inline ncdiff/src/yang/ncdiff/runningconfig.py:105]: Could this line be:
`    (re.compile(r'^ *ospfv3 \d+ neighbor '), 1),`

File: ncdiff/src/yang/ncdiff/runningconfig.py (hunk 1 above)
Comment (sowmyadn010501) [reply ncdiff/src/yang/ncdiff/runningconfig.py]: Yeah got it. Thank you. I have updated.

File: ncdiff/src/yang/ncdiff/runningconfig.py (hunk 2 above)
Comment (sowmyadn010501) [reply ncdiff/src/yang/ncdiff/runningconfig.py]: Updated

**Review by yuekyang — APPROVED** [review]

**Review by Taarini — APPROVED** [review]

**Pull request merged**